### DISCUSSION TITLE
Refine by Site Title on Document Search Results

### DISCUSSION THREAD

**Discussion by JonoSuave** [microsoft-search/pnp-modern-search#2032] [Q&A] [answered]
Is there a way to refine by SiteTitle for search results that query only documents? I don't believe SiteTitle is refinable so I mapped the crawled property ows_SiteName to RefinableString00. I believe that's working okay. The issue is that I don't believe that files have any metadata that shows the site name it belongs to.

**Comment by simesy** [comment, marked as answer]
This are for a Document Library that only contains Office documents. Works well for me just with ows_SiteName - complete coincidence that we both are mapping  ows_SiteName to RefinableString00.

<img width="1225" alt="Screen Shot 2022-04-22 at 10 53 01 pm" src="https://user-images.githubusercontent.com/188856/164718604-d01003de-0440-4405-90a8-595a7bb9a46c.png">
<img width="620" alt="Screen Shot 2022-04-22 at 10 52 49 pm" src="https://user-images.githubusercontent.com/188856/164718606-44977323-b731-4059-9f49-7dfc83e738ce.png">
<img width="1053" alt="Screen Shot 2022-04-22 at 10 54 54 pm" src="https://user-images.githubusercontent.com/188856/164718599-0b27601b-6796-40ab-af4b-2b93125804ed.png">

**Reply by wobba** [reply]
SiteTitle should be present on all items in a site.

**Reply by wobba** [reply]
But of course SiteTitle is not refinable so you need to remap it 🙂

**Comment by simesy** [comment]
I've been having inconsistent results and I think this is the best way to do it. https://sharepoint.stackexchange.com/a/110165/102716
